net-dns/inadyn/inadyn-2.12.0-r3.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools systemd tmpfiles
+
+DESCRIPTION="Dynamic DNS client with multiple SSL/TLS library support"
+HOMEPAGE="https://github.com/troglobit/inadyn"
+SRC_URI="https://github.com/troglobit/inadyn/releases/download/v${PV}/${P}.tar.xz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~mips ~ppc ~ppc64 ~riscv ~sparc ~x86"
+IUSE="gnutls mbedtls openssl"
+REQUIRED_USE="?? ( gnutls mbedtls openssl )"
+
+DEPEND="
+	acct-group/inadyn
+	acct-user/inadyn
+	dev-libs/confuse:=
+	gnutls? (
+		dev-libs/nettle:=
+		net-libs/gnutls:=
+	)
+	mbedtls? ( net-libs/mbedtls:3= )
+	openssl? ( dev-libs/openssl:= )
+"
+RDEPEND="${DEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+PATCHES=( "${FILESDIR}/${PN}-2.12.0-musl.patch" )
+
+src_prepare() {
+	default
+	# Adjust mbedtls pkg-config module
+	sed -i \
```

Comment:
I would prefer this as a patch, given that could:
a) go upstream;
b) won't silently stop doing anything.

Reply:
Yes, I also think, a patch would be better. Could you submit this to upstream? They seem pretty active.

Reply:
@winterheart ping